templates.go

```diff
@@ -1196,10 +1200,10 @@ coreos:
       --cluster-domain={{.Cluster.Kubernetes.Domain}} \
       --network-plugin=cni \
       --register-node=true \
-      --register-schedulable=false \
+      --register-with-taints=node-role.kubernetes.io/master=:NoSchedule \
```

Comment:
Why we need to add new label? Is it enough to add `role=master:NoSchedule`?

Reply:
This label is used in community and components like Calico rely on this label. See links in description